src/ops/convolution.cc

```diff
@@ -0,0 +1,230 @@
+/*******************************************************************************
+* Copyright 2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+#include <vector>
+
+#include "../../../../src/operator/nn/convolution-inl.h"
+#include "convolution.h"
+
+#include <ngraph/builder/quantization.hpp>
+#include "../ngraph_emitter.h"
+#include "../ngraph_emitter_utils.h"
+#include "../ngraph_nnvm_ops.h"
+#include "../ngraph_sgcompiler_utils.h"
+#include "../ngraph_utils.h"
+
+#if MXNET_USE_MKLDNN == 1
+#include "../../../../src/operator/subgraph/mkldnn/mkldnn_conv-inl.h"
+#endif
+
+namespace ngraph_bridge {
+
+struct ConvInputs {
+  NgraphNodePtr data;
+  NgraphNodePtr filter;
+  NgraphNodePtr bias;
+  ngraph::CoordinateDiff pad;
+  ngraph::Strides stride;
+  ngraph::Strides dilate;
+  size_t groups;
+};
+
+ConvInputs get_conv_inputs(NgraphNodePtr data, NgraphNodePtr filter,
+                           NgraphNodePtr bias,
+                           const mxnet::op::ConvolutionParam& param) {
+  ConvInputs conv_inputs;
+  conv_inputs.data = data;
+  conv_inputs.filter = filter;
+  conv_inputs.bias = bias;
+  conv_inputs.pad = ngraph::CoordinateDiff{param.pad.begin(), param.pad.end()};
+  conv_inputs.stride =
+      ngraph::Strides{param.stride.begin(), param.stride.end()};
+  conv_inputs.dilate =
+      ngraph::Strides{param.dilate.begin(), param.dilate.end()};
+  conv_inputs.groups = param.num_group;
+  return conv_inputs;
+}
+ConvInputs get_conv_inputs(Emitter* emitter, const NodePtr& node,
+                           const mxnet::op::ConvolutionParam& param) {
+  auto data = emitter->op_map_[node->inputs_[0]];
+  auto filter = emitter->op_map_[node->inputs_[1]];
+  NgraphNodePtr bias =
+      param.no_bias ? nullptr : emitter->op_map_[node->inputs_[2]];
+  return get_conv_inputs(data, filter, bias, param);
+}
+
+NgraphNodePtr create_convolution(const ConvInputs& conv_inputs) {
```

Comment:
If this overload of the function is only used within this TU, would it make sense to make it `static`?  It might help readers understand the code more quickly.